doc/source/structures/collections/list.rst

```diff
@@ -41,6 +41,9 @@ Structure
           - Type
           - Description
 
+        * - All suffixes of :struct:`Enumerable`
+          -
+          - :struct:`List` objects are a type of :struct:`Enumerable`
```

Comment:
Man I should really look into whether or not Sphinx has a sort of include operator that can be used for cases like this.  We have a lot of them.